tests/publiccloud/run_ltp.pm

```diff
@@ -24,6 +24,11 @@ use version_utils;
 
 our $root_dir = '/root';
 
+# LTP runtime can be 'python' (new runner) or the default old 'perl' runner
+my $ltp_runtime = get_var('LTP_RUNTIME','perl');
+
+die ('Not supported LTP_RUNTIME provided') unless ( $ltp_runtime =~ /^(python|perl)/i );
```

Comment:
This to allow 2 only values as runtime, for https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/15583#discussion_r987826773

Reply:
While it works this is suboptimal style because you now have two places where you check for those variables.

Assuming you would add a third option tomorrow, e.g. `bash`, you would need to add it now it to places: here and in the if block. If you just add it to the if block, you only have one place to update and it is clear that you have to do it.

Reply:
I agree with you if **one** only switch case is present, while I don't think that more than 2 options can happen, otherwise it means to have 3 or more contemporary different _languages_ in LTP projects.
My consideration was that without this initialization, in the case more future switch case in the code should be needed, additional 'elsif' means a more test to add instead of simple 'else' and more complexity. 
Anyway I will update the code as by you suggestion.